nautobot_firewall_models/models/security_policy.py

```diff
@@ -93,8 +94,8 @@ class PolicyRule(PrimaryModel):
     application_groups = models.ManyToManyField(
         blank=True, to="nautobot_firewall_models.ApplicationObjectGroup", related_name="policy_rules"
     )
-    request_id = models.CharField(max_length=100, blank=True)
-    description = models.CharField(max_length=200, blank=True)
+    request_id = models.CharField(max_length=CHARFIELD_MAX_LENGTH, blank=True)
+    description = models.CharField(max_length=1024, blank=True)
```

Comment:
Is there a reason we're using 1024 instead of CHARFIELD_MAX_LENGTH here?

Reply:
To support Juniper and Checkpoint's description field limit length.

This was the PR https://github.com/nautobot/nautobot-app-firewall-models/pull/327
And the associated issue https://github.com/nautobot/nautobot-app-firewall-models/issues/295

Reply:
CHARFIELD_MAX_LENGTH is just our default value for `Charfield.max_length`